### PR TITLE
Upgrade to pulumi-terraform-bridge v2.11.0

### PR DESCRIPTION
**Pull request by stack72**

### CODE REVIEW

File: sdk/nodejs/s3/bucket.ts [hunk 1]
```diff
@@ -7,7 +7,7 @@ import * as outputs from "../types/output";
 import * as utilities from "../utilities";
 
 import {PolicyDocument} from "../iam";
-import {CannedAcl} from "./index";
+import {CannedAcl, RoutingRule} from "./index";
```
Comment (komalali) [inline sdk/nodejs/s3/bucket.ts:10]: This is strange. It doesn't look like `RoutingRule` is used in this file at all.

Comment (stack72) [reply sdk/nodejs/s3/bucket.ts:10]: 🤷 

This is jsut what was generated locally for me

Comment (komalali) [reply sdk/nodejs/s3/bucket.ts:10]: Yeah I know I'm seeing it too in the enums PR. It's not a big deal really, mostly just curious why it's happening.

Interesting though that your machine didn't have whatever issues mine is having with regressions related to other providers. But glad it's working for you at least.

Comment (stack72) [reply sdk/nodejs/s3/bucket.ts:10]: Welcome to the world of provider generation :D

**Review by komalali — APPROVED** [review]

**Pull request merged**